doc/ref_model/chapters/chapter07.rst

```diff
@@ -1707,79 +1721,3 @@ Compliance with Standards
 
 **Table 7-16:** Compliance with standards requirements
 
-Security References
```

Comment:
@karinesevilla Instead of deleting can we refer to them as "Additional Security References"? It makes this a go-to place.

Reply:
@pgoyal01 @walterkozlowski Okay , it can be kept for information

Reply:
Done